app/models/procedure.rb

```diff
@@ -690,9 +690,11 @@ def instructeurs_self_management?
     routee? || instructeurs_self_management_enabled?
   end
 
-  def defaut_groupe_instructeur_for_new_dossier
+  def groupe_instructeur_for_new_dossier
     if !routee? || feature_enabled?(:procedure_routage_api)
       defaut_groupe_instructeur
+    elsif routee? && self.groupe_instructeurs.size == 1
+      self.groupe_instructeurs.first
```

Comment:
```suggestion
      defaut_groupe_instructeur
```

`defaut_groupe_instructeur` c'est un alias vers `groupe_instructeurs.first`